packages/web-components/src/components/cbp-notice/cbp-notice.stories.tsx

```diff
@@ -0,0 +1,54 @@
+export default {
+    title: 'Components/Notice',
+    tags: ['autodocs'],
+    argTypes: {
+      label: {
```

Comment:
The Label control is not being used. Hoist the title and content up here in argTypes and name them with (slotted) so it's clear they aren't mapping to props.

Reply:
updated